### PR TITLE
Use Scala 2.11.8 on Travis CI

### PR DESCRIPTION
**Pull request by rexim**
### Description

I compared Morganey with my [other scala project](https://github.com/tsoding/Issuestant/blob/master/.travis.yml) that I setup recently. And everything works fine there. So I tried to make Morganey's travis config closer to the Issuestant's one. Looks like it helped.

### CODE REVIEW

**Comment by keddelzz** [comment]
LGTM :+1:

**Comment by rexim** [comment]
Even though the build for #193 is fixed without this PR, I think it still better to keep the scala version in `.travis.yml` and `build.sbt` in sync.

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 language: scala
-jdk: oraclejdk7
-sudo: false
```
Comment (rexim) [inline .travis.yml:3]: Also, I'm not sure if it was a good idea to get rid of these two. Let me get them back and see if it doesn't break anything.

**Comment by rexim** [comment]
Hm... looks like Mr @ForNeVeR returned back to his cave. Alright! Not waiting for him then!

**Comment by ForNeVeR** [comment]
> back to his cave

I told you, it's not the cave but the [kathisma](https://en.wikipedia.org/wiki/Kathisma#Monastic_cell) where I'm praying the Machine God, executing important rituals, and sometimes — for sure! — degustating the best canaries of machine oil.

And, for that matter, I approve this PR. LGTM 👍

**Comment by rexim** [comment]
Thank you very much @ForNeVeR! :laughing:

**Pull request merged**